### ISSUE TITLE
Multiple comparisons adjustments, null control etc. in bayestestR

### ISSUE THREAD

**Issue by DominiqueMakowski** [easystats/bayestestR#485]
In the debate whether yes or no Bayesian analyses could/should be "corrected" for multiple testing, comparisons and false positives, this suggests extending the null control method (implemented in JASP):

[Null_Control_Jong.pdf](https://github.com/easystats/bayestestR/files/7489406/Thesis_Tim.de.Jong_Final.version.pdf)

It would be nice to make something like that available in bayestestR

Could interest @mattansb

**Comment by mattansb** [comment]
These "adjustments" are for the **prior *odds*** between pairs/sets of hypotheses (not for any of the measures of evidence).
How would you like to see these implemented?

(Personally not a fan of these "let *me* tell you what *your* priors are" tools 😅)

**Comment by mattansb** [comment]
I don't think this is an avenue worth perusing (see previous comment about defining others' priors).

But if *you* are interested in such methods of default priors, you might also want to take a look at the `BAS` package, which computes term-wise adjusted priors.

**Comment by strengejacke** [comment]
> (Personally not a fan of these "let _me_ tell you what _your_ priors are" tools 😅)

![image](https://user-images.githubusercontent.com/26301769/157860410-8a844cdc-1e0c-4da9-975c-ab0e03c8c57b.png)

**Comment by bwiernik** [comment]
We do also have curve-wise intervals, so that is one method for multiple testing adjustment

**Comment by mattansb** [comment]
Yup, I'm def more in favor of those.
(The ones discussed here are for independent tests, or pairwise model-less tests...)
